addons/analytic/static/src/components/analytic_distribution/analytic_distribution.xml

```diff
@@ -51,7 +51,7 @@
                                         class="o_input"
                                         inputmode="numeric"
                                         type="text"
-                                        t-att-value="formatPercentage(dist_tag.percentage)"
+                                        t-att-value="dist_tag.percentage ? formatPercentage(dist_tag.percentage) : ''"
```

Comment:
Spec:
> Hide 0% when no account is set and percentage left is 0%

Here we hide it if there is an account with 0%. Try selecting multiple accounts using the *View All* option - is that the result we want? If so, ignore this, otherwise check the `dist_tag.analytic_account_id`